10-gis.Rmd

```diff
@@ -606,54 +623,6 @@ Use `"v.select"` and `"v.extract"` for vector data.
 - You can also start R from within a running GRASS\index{GRASS} session [for more information please refer to @bivand_applied_2013].
 - Refer to the excellent [GRASS online help](https://grass.osgeo.org/grass82/manuals/) or `execGRASS("g.manual", flags = "i")` for more information on each available GRASS geoalgorithm\index{geoalgorithm}.
```

Comment:
```suggestion
- Refer to the excellent [GRASS online help](https://grass.osgeo.org/grass-stable/manuals/) or `execGRASS("g.manual", flags = "i")` for more information on each available GRASS geoalgorithm\index{geoalgorithm}.
```

Reply:
Thanks a lot for the suggested changes @neteler I will merge them.